boards/px4/fmu-v6x/src/manifest.c

```diff
@@ -142,8 +163,12 @@ static px4_hw_mft_list_entry_t mft_lists[] = {
 	{V6X00, hw_mft_list_v0600, arraySize(hw_mft_list_v0600)},
 	{V6X01, hw_mft_list_v0600, arraySize(hw_mft_list_v0600)}, // BMP388 moved to I2C2
 	{V6X03, hw_mft_list_v0600, arraySize(hw_mft_list_v0600)}, // BMP388 moved to I2C2, Sensor Set 3
-	{V6X50, hw_mft_list_v0650, arraySize(hw_mft_list_v0650)}, //                       HB Mini
-	{V6X51, hw_mft_list_v0650, arraySize(hw_mft_list_v0650)}, // BMP388 moved to I2C2  HB Mini
+	//{V6X40, hw_mft_list_v0640, arraySize(hw_mft_list_v0640)}, //                       HB CM4 base           // never shipped
+	//{V6X41, hw_mft_list_v0640, arraySize(hw_mft_list_v0640)}, // BMP388 moved to I2C2  HB CM4 base           // never shipped
+	{V6X43, hw_mft_list_v0640, arraySize(hw_mft_list_v0640)}, // BMP388 moved to I2C2, HB CM4 base Sensor Set 3
+	{V6X44, hw_mft_list_v0640, arraySize(hw_mft_list_v0640)}, // BMP388 moved to I2C2, HB CM4 base Sensor Set 4
+	//{V6X50, hw_mft_list_v0650, arraySize(hw_mft_list_v0650)}, //                       HB Mini               // never shipped
+	//{V6X51, hw_mft_list_v0650, arraySize(hw_mft_list_v0650)}, // BMP388 moved to I2C2  HB Mini               // never shipped
```

Comment:
```suggestion
	{V6X40, hw_mft_list_v0600, arraySize(hw_mft_list_v0600)}, //                       HB CM4 base                    RC01 FMUM
	{V6X41, hw_mft_list_v0600, arraySize(hw_mft_list_v0600)}, // BMP388 moved to I2C2  HB CM4 base  RC01 FMUM
	{V6X43, hw_mft_list_v0600, arraySize(hw_mft_list_v0600)}, // BMP388 moved to I2C2, HB CM4 base Sensor Set 3
	{V6X44, hw_mft_list_v0640, arraySize(hw_mft_list_v0600)}, // BMP388 moved to I2C2, HB CM4 base Sensor Set 4
	{V6X50, hw_mft_list_v0650, arraySize(hw_mft_list_v0650)}, //                       HB Mini                    RC01 FMUM
	{V6X51, hw_mft_list_v0650, arraySize(hw_mft_list_v0650)}, // BMP388 moved to I2C2  HB Mini  RC01 FMUM
```

Reply:
The two changes combined seem to only increase flash by 40 bytes or so, so that should be fine.

Reply:
@davids5 this suggests to add V6X51 back in but it's not listed in the initSPI below. Does that make sense?

Reply:
@nicovanduijn do have any revs greater then RC01 of the V6?

Reply:
not yet, but we might in the near future. Is there a community table for the rev and vers for v6x? We should probably also reserve one resistor each to denote " read from EEPROM ". On v5x we have that for ver, but not yet for rev if I'm not mistaken